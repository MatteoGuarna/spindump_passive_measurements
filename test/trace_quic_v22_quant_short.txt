First simple try with draft 22. Using Quant.
