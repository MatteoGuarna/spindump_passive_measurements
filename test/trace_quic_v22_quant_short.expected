[
{ "Event": "new", "Type": "QUIC", "Addrs": ["31.133.154.150","52.58.13.57"], "Session": "041f3b3a-94a9cd6ae1c626b354dc (58579:4433)", "Ts": 1563659727418694, "State": "Starting", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "change", "Type": "QUIC", "Addrs": ["31.133.154.150","52.58.13.57"], "Session": "041f3b3a-081a6f7cf1b3c3ed (58579:4433)", "Ts": 1563659727539565, "State": "Starting", "Packets1": 1, "Packets2": 0, "Bytes1": 1228, "Bytes2": 0, "Bandwidth1": 1228, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["31.133.154.150","52.58.13.57"], "Session": "041f3b3a-081a6f7cf1b3c3ed (58579:4433)", "Ts": 1563659727539565, "State": "Up", "Right_rtt": 120871, "Packets1": 1, "Packets2": 0, "Bytes1": 1228, "Bytes2": 0, "Bandwidth1": 1228, "Bandwidth2": 0 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["31.133.154.150","52.58.13.57"], "Session": "1f3b3a13-081a6f7cf1b3c3ed (58579:4433)", "Ts": 1563659727539573, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 1, "Packets2": 1, "Bytes1": 1228, "Bytes2": 421, "Bandwidth1": 1228, "Bandwidth2": 421 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["31.133.154.150","52.58.13.57"], "Session": "1f3b3a13-081a6f7cf1b3c3ed (58579:4433)", "Ts": 1563659727539575, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 1, "Packets2": 2, "Bytes1": 1228, "Bytes2": 752, "Bandwidth1": 1228, "Bandwidth2": 752 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["31.133.154.150","52.58.13.57"], "Session": "1f3b3a13-081a6f7cf1b3c3ed (58579:4433)", "Ts": 1563659727545739, "State": "Up", "Value": 1, "Who": "initiator", "Packets1": 2, "Packets2": 3, "Bytes1": 1372, "Bytes2": 1344, "Bandwidth1": 1372, "Bandwidth2": 1344 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["31.133.154.150","52.58.13.57"], "Session": "1f3b3a13-081a6f7cf1b3c3ed (58579:4433)", "Ts": 1563659727546360, "State": "Up", "Value": 0, "Who": "initiator", "Packets1": 3, "Packets2": 3, "Bytes1": 1455, "Bytes2": 1344, "Bandwidth1": 1455, "Bandwidth2": 1344 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["31.133.154.150","52.58.13.57"], "Session": "d2c78292-081a6f7cf1b3c3ed (58579:4433)", "Ts": 1563659727666766, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 4, "Packets2": 4, "Bytes1": 1512, "Bytes2": 1418, "Bandwidth1": 1512, "Bandwidth2": 1418 }
]
